src/pages/guides/using/marketo/marketo-data-streams.md

```diff
@@ -0,0 +1,82 @@
+
+Note: The Marketo Data Streams are currently in Beta
+
+## Subscribing to Marketo Events
+
+See the [Adobe I/O Events Docs](/src/pages/index.md)
+
+Here are some basic getting started instructions that will be the same for each of the available Marketo data streams in I/O Events.
+
+- Start from the [developer console](/console/):
+
+- Select `Create new project`
+
+  ![Create new project](../../img/UserAuditDataStreamIOSetup1.png "Quick Start")
+
+- Select `Add event`
+
+  ![Add event](../../img/UserAuditDataStreamIOSetup2.png "Get started with your new project by adding an event subscription")
+
+- Filter by `Experience Cloud`
+
+You should see the three Marketo Data Streams available.  Please see the individual page for each stream for further setup instructions.
+
+## Developer Guidelines
+
+When setting up a project to subscribe to events, there are three ways to interact with those event subscriptions in order to receive the events.  The first is Journaling, which provides a pull model in which events can be pulled via API and stores up to 7 days of past events.  The second is Webhooks, which can be configured to send events either as single events or batched to a webhook endpoint in near real-time with the event occurrence.  Third is Runtime, where you can set up your own custom function within Adobe that events will automatically run through near-real time.
+
+### Journaling
+[Getting Started with Journaling](../../journaling_intro.md)
+
+Important Takeaways:
+
+- Stores up to 7 days of history
+- Can be iterated through from any previous event within the history
+- Will still receive and store events even if webhook is failing
+- Useful for fetching events that were missed due to webhook issues or for a pulling mechanism instead of webhook push
+
+### Webhooks
+[Getting Started with Adobe I/O Events Webhooks](../../index.md)
+
+Webhook Endpoint Requirements:
+
+- Handle GET and POST requests
+- Respond with a 200-type response within a reasonable time period
+- Challenge Request
+  - GET request with challenge query parameter
+  - Must respond with value of challenge query parameter
+- Webhook Events
+  - POST request with JSON data body with one or more events
+  - *Recommended to set up webhook as batch*
+
+## Debug
+
+[Debug Tracing](../../../support/tracing.md)
+
+Once you have successfully completed your setup and event subscription registration, events should start being stored in the journal.  In addition, if you have webhooks or runtime set up, the events will go through those flows.  From the project's page in the event registration details, you should see a tab for Debug Tracing.  For webhooks, this will show a record of failed and successful challenge attempts as well as webhook attempts.  Each request includes the request/response details to help debug.
+
+There is also an Event Browser tab which lets you manually look at events in the Journal from the Developer Console UI
```

Comment:
I see all of these sections have been duplicated in your guides. You could always create a reusable component and include that in all your guides. See how we have done this for one of our common snippets:
1. [Add a common doc](https://github.com/AdobeDocs/adobe-io-events/blob/main/src/pages/common/retry-doc.md)
2. [Add a declaration on the top of the page where the common snippet needs to be inserted](https://github.com/AdobeDocs/adobe-io-events/blob/main/src/pages/support/faq.md?plain=1#L5C1-L5C1)
3. [Simply use it](https://github.com/AdobeDocs/adobe-io-events/blob/main/src/pages/support/faq.md?plain=1#L81)

Reply:
Cool didn't realize we could do that.  @ray-eckard we can probably pull the Developer Guidelines, Journaling, and Webhooks sections out to a separate page as well as the debug section as it's own separate page.  We might also be able to pull out a portion of subscription instructions section as well as there's some duplication, though that's probably less important since it's only partially duplicated